docs/spec.md

```diff
@@ -1531,18 +1537,22 @@ strict lower triangle correspondingly, are implementation-defined.
 If there exists `i` where the input matrix is not an Hermitian positive-definite
 matrix, then the behavior is undefined.
 
+For quantized types, performs
+`quantize(cholesky(dequantize(a), lower), type(result))`.
```

Comment:
`quantize_op_dequantize(lambda operand: cholesky(operand, lower), operand, type(result))`. Let's centralize the quantize/dequantize logic in a single function. Same everywhere below.

Reply:
done.